projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/files/entrypoint/run_ilab.sh

```
@@ -95,11 +95,17 @@ ret=0
 if ! torchrun \
     --node_rank "${RANK}" \
     --rdzv_endpoint "${MASTER_ADDR}:${MASTER_PORT}" \
+    ./opt/app-root/lib/python3.11/site-packages/instructlab/training/main_ds.py \
     $(echo "$config_json" | jq -r '. | to_entries | .[] | ("--" + .key + " " + (.value | tostring))' | sed "s/ true//");
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)